packages/framework/client-logger/fluid-telemetry/README.md

```diff
@@ -112,42 +102,23 @@ if (containerExists) {
     myAppContainerId = await myAppContainer.attach();
 }
 
-// 2: This is our implementation of ITelemetryConsumer that will send telemetry to Azure App Insights
-class AppInsightsTelemetryConsumer implements ITelemetryConsumer {
-    constructor(private readonly appInsightsClient: ApplicationInsights) {}
-
+// 2: This is our implementation of ITelemetryConsumer
```

Comment:
Probably remove section 2 here, since Step 2 above has this same class definition?

Reply:
I see how that might be confusing. I've updated the example to be as if you're imported the implementation from another file.